src/core/ext/filters/client_idle/idle_filter_state.cc

```diff
@@ -18,6 +18,8 @@
 
 #include <assert.h>
 
+#include <stdint>
```

Comment:
This looks unrelated to the rest of the PR.  Why is this needed here?

Reply:
Oops, I didn't mean to include this, I was having unrelated build errors.  This file was the first the build broke on but it seems to be something wrong with my bazel setup (or building internal to google) since other files also had warnings.

$ bazel test //test/cpp/end2end:client_lb_end2end_test
...
ERROR: /usr/local/google/home/samuelw/grpc/BUILD:1688:16: Compiling src/core/lib/event_engine/sockaddr.cc failed: undeclared inclusion(s) in rule '//:grpc_sockaddr':
this rule is missing dependency declarations for the following files included by 'src/core/lib/event_engine/sockaddr.cc':
  '/usr/lib/clang/13.0.1/include/stdint.h'